.docker/scripts/init-ext.sh

```diff
@@ -7,7 +7,7 @@ set -e
 . /app/ckan/default/bin/activate
 
 pip install -r "/app/requirements.txt"
-pip install -r "/app/dev-requirements.txt"
```

Comment:
renamed `dev-requirements.txt` to `requirements-dev.txt` since it is more commonly used way of defining dev requirements.